src/components/Footer/Footer.tsx

```diff
@@ -23,6 +23,7 @@ const Footer: React.FC = () => {
               <Flag name="china" size="md" />
               <Flag name="austria" size="md" />
               <Flag name="germany" size="md" />
+              <Flag name="ukraine" size="md" />
```

Comment:
Sorry, but this should not be part of this PR

Reply:
Yes, as @priscilawebdev suggests. The footer flag is like a memorial place for those that settled the bases of this project. It's a reserved area and has to be earned with contributions during a long period of time (it's a non-written rule), it just happened. 

The current ones represent the countries where live / born the most active maintainers. 👍 . The latest one we added was Germany (my new home), looking forward to adding new flags.

Just wanna give you more context so don't get it wrong.

Reply:
++